public/docs/ts/latest/guide/user-input.jade

```diff
@@ -150,76 +148,67 @@ figure.image-display
+  If the user mouses away and clicks elsewhere on the page,
+  the previous example won't transfer the current state of the input box.
+  You update the component's `values` property only when the user presses Enter
   while the focus is inside the input box.
 
-  Let's fix that by listening to the input box's blur event as well.
+  To fix that, listen to the input box's blur event as well.
 
 +makeExample('user-input/ts/app/keyup.components.ts', 'key-up-component-4' ,'app/keyup.components.ts (v4)')(format=".")
 
 .l-main-section
 :marked
   ## Put it all together
-  We learned how to [display data](./displaying-data.html) in the previous chapter.
-  We've acquired a small arsenal of event binding techniques in this chapter.
+  The previous chapter showed how to [display data](./displaying-data.html).
+  This chapter demonstrated event binding techniques.
 
-  Let's put it all together in a micro-app
+  Now, put it all together in a micro-app
   that can display a list of heroes and add new heroes to that list.
-  The user can add a hero by first typing in the input box and then
-  pressing Enter, clicking the Add button, or clicking elsewhere on the page.
+  The user can add a hero by typing the hero's name in the input box and then
+  clicking **Add**.
 
 figure.image-display
     img(src='/resources/images/devguide/user-input/little-tour-anim.gif' alt="Little Tour of Heroes")
 :marked
   Below is the "Little Tour of Heroes"  component.
-  We'll call out the highlights after we bask briefly in its minimalist glory.
 
 +makeExample('user-input/ts/app/little-tour.component.ts', 'little-tour', 'app/little-tour.component.ts')(format=".")
 :marked
-  We've seen almost everything here before. A few things are new or bear repeating.
 
   ### Use template variables to refer to elements
 
   The `newHero` template variable refers to the `<input>` element.
-  We can use `newHero` from any sibling or child of the `<input>` element.
+  You can use `newHero` from any sibling or child of the `<input>` element.
 
   Getting the element from a template variable makes the button click handler
-  simpler. Without the variable, we'd have to use a fancy CSS selector
+  simpler; without the variable, you would have to use a CSS selector
   to find the input element.
 
   ### Pass values, not elements
 
-  We could have passed the `newHero` into the component's `addHero` method.
-
-  But that would require `addHero` to pick its way through the `<input>` DOM element,
-  something we learned to dislike in our first try at a [keyup component](#keyup1).
-
-  Instead, we grab the input box *value* and pass *that* to `addHero`.
-  The component knows nothing about HTML or the DOM, which is the way we like it.
+  Instead of passing the `newHero` into the component's `addHero` method,
+  grab the input box *value* and pass *that* to `addHero`.
 
   ### Keep template statements simple
   We bound `(blur)` to *two* JavaScript statements.
```

Comment:
In the section that follows, there are lots of "we" and some sentences similar to this one: "The second statement exists for a good reason" that could be cut or minimized.

This paragraph, "Although the example works, we are rightly wary of JavaScript in HTML. Template statements are powerful. We're supposed to use them responsibly. Complex JavaScript in HTML is irresponsible" and "Should we reconsider our reluctance to pass the input box into the component?", could be reworked. 

How about something along these lines:

"This example works and is fine for a demonstration, but a better practice is to separate the JavaScript and HTML. Refactor using NgModel as covered in the Forms page."